tests/phpunit/CRM/Member/Form/MembershipRenewalTest.php

```diff
@@ -89,6 +89,7 @@ public function setUp() {
       'fixed_period_start_day' => '101',
       'fixed_period_rollover_day' => '1231',
       'relationship_type_id' => 20,
+      'min_fee' => 100,
```

Comment:
Actually this didn't seem to 'work' & the underlying is still set to 0 - since we are testing overrides that is OK for now